### ISSUE TITLE
Ticket Sale: Floating Action button for buying tickets or viewing already bought tickets 

### ISSUE THREAD

**Issue by velev8** [TCA-Team/TumCampusApp#910]
- button should be an obvious color
- include price in the event details
- link shuld be as a separate button below so the picture is bigger

**Comment by saileru** [comment]
@kordianbruck Can we close this issue?

I think we already made greate improvements to the event details fragment.
Button should be obvious colour. Check!
![screenshot_20180707-132815](https://user-images.githubusercontent.com/12066533/42410508-0bde17f2-81eb-11e8-8dfd-181274849481.png)

Link shuld be as a separate button below so the picture is bigger.
I think we also did that. The link to the website is now below the description.
![screenshot_20180707-133948](https://user-images.githubusercontent.com/12066533/42410529-57307cc2-81eb-11e8-98e0-d03b2bed4a34.png)

Is it really necessary to include the price in the event details? Currently, it is displayed in the BuyTicketActivity after the users presses the "Buy Now button". This should be sufficient, right? Also, it is not clear which price to display in the event details when we have multiple ticket types!
